specificatie/openapi.yaml

```diff
@@ -326,10 +326,10 @@ components:
             $ref: 'verblijfstitel.yaml#/components/schemas/Verblijfstitel'
         geheimhoudingPersoonsgegevens:
           $ref: 'persoon.yaml#/components/schemas/GeheimhoudingPersoonsgegevens'
-        verificatie:
-          $ref: 'common.yaml#/components/schemas/Verificatie'
         opschortingBijhouding:
           $ref: 'opschortingbijhouding.yaml#/components/schemas/OpschortingBijhouding'
+        verificatie:
+          $ref: 'common.yaml#/components/schemas/GbaVerificatie'
```

Comment:
iets te snel gedaan. Dit is de GBA variant in de niet-GBA operapi.yaml

Reply:
Inderdaad iets te snel willen doen. Nu aangepast.